src/certificates/with-modes-without-certificates/WithModesWithoutCertificates.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Button, Stack } from '@edx/paragon';
+import { Add as AddIcon } from '@edx/paragon/icons';
+
+import MainLayout from '../layout/MainLayout';
+import messages from '../messages';
+
+const WithModesWithoutCertificates = ({ courseId }) => {
+  const intl = useIntl();
+  return (
+    <MainLayout showHeaderButtons={false} courseId={courseId}>
+      <Stack direction="horizontal" className="p-5 justify-content-center align-items-center bg-light-300" gap="3.5">
+        <span>{intl.formatMessage(messages.noCertificatesText)}</span>
+        <Button
+          alt={intl.formatMessage(messages.setupCertificateBtnAlt)}
```

Comment:
[important] Since we are displaying a decorative image inside a button, we do not need to specify `alt` for it. In our case, the screen reader will tell the user that this is a button and its text.

Reply:
Thanks, i`ll remove it!